src/util/cnames.js

```diff
@@ -75,6 +75,8 @@ const parseCNAMEsFile = (content, context = {}) => {
 
         // Convert vercel.app to cname.vercel-dns.com
         if (target.endsWith('.vercel.app')) target = 'cname.vercel-dns.com';
+        // Convert netlify.com to netlify.app
```

Comment:
```suggestion

        // Convert netlify.com to netlify.app
```
Nitpick, empty line between blocks of logic